### ISSUE TITLE
Getting errors on prepare.py

### ISSUE THREAD

**Issue by omgnuts** [BelledonneCommunications/linphone-desktop#1]
Hello, I tried ./prepare.py -c & ./prepare.py on MacOS. Getting errors on the make as follows:

```
Traceback (most recent call last):
  File "./prepare.py", line 263, in <module>
    sys.exit(main())
  File "./prepare.py", line 247, in main
    if target.generator.endswith('Ninja'):
AttributeError: 'NoneType' object has no attribute 'endswith'
```

**Comment by ghismary** [comment]
Hi,
This has already been fixed with revision 354b175a6b584f47bae5d5ace80342d1b31d1fc7.

**Comment by ghismary** [comment]
Sorry there was still an issue, it should really be fixed now with c9412d35fbc5d2f5df9a2d656aed6455d7c81a29.
